umn/source/model_templates/template_description/pytorch-py37_general_template.rst

```diff
@@ -2,13 +2,13 @@
 
 .. _modelarts_23_0166:
 
-PyTorch-py36 General Template
+PyTorch-py37 General Template
 =============================
 
 Introduction
 ------------
 
-AI engine: PyTorch 1.0; Environment: Python 3.6; Input and output mode: undefined mode. Select an appropriate input and output mode based on the model function or application scenario. When using the template to import a model, select the **model** directory containing the model files.
+AI engine: PyTorch 1.0; Environment: Python 3.7; Input and output mode: undefined mode. Select an appropriate input and output mode based on the model function or application scenario. When using the template to import a model, select the **model** directory containing\ :ref:`Specifications for Compiling Model Inference Code <modelarts_23_0093>` the model files.
```

Comment:
Where should have a space? Did you fix it already? If yes, thanks a lot.

Reply:
I have not and not going to since it comes from the source:

https://github.com/opentelekomcloud-docs/doc-exports/blob/2e98ed0977be6110cf6449947a14d896cc28bfa7/docs/modelarts/umn/modelarts_23_0166.html#L4 contains `</strong> directory containing<a href="modelarts_23_0093.html">Specifications for Compiling `. This results exactly in what you see here: `directory containing\ :ref:`Specifications `.

Proper fix is to add missing space in the original html (and this should be done in the source, and not simply updating the PR

Reply:
I am not sure if they can fix in the html file. If they provide us a new html file, we need to create a new PR. Should we?

Reply:
I would vote for the option a) since we need time to start the content review of the whole UMN in detail. 
Meanwhile, I can contact our vendor to fix specific issues which we can not fix.